v2/cmd/config/checks.go

```diff
@@ -81,6 +81,14 @@ func setCheckSBOMClient(check voucher.Check, gcrClient voucher.SBOMClient) {
 	}
 }
 
+func setCheckSBOMVulnerabilityClient(check voucher.Check, gcrClient voucher.SBOMClient, vulnFailList []string, severity string) {
+	if sbomVulnerbilityCheck, ok := check.(voucher.SbomVulnerabilityCheck); ok {
+		sbomVulnerbilityCheck.SetSBOMClient(gcrClient)
```

Comment:
```suggestion
```

I think we can skip this, because we call `setCheckSBOMClient(check, sbom)` right before `setCheckSBOMVulnerabilityClient`, and it casts to the embedded `SbomClientCheck` interface.

Reply:
Neat!  that was a good catch with the inheritance!

Reply:
resolved in commit 4e267e9